src/h3_hexagons/nso_tif_model_iterater_hexagon_output.py

```diff
@@ -0,0 +1,73 @@
+import h3
+import pyproj
+import pandas as pd
+import geopandas as gpd
+from shapely.geometry import Polygon
+
+
+def output_h3_hexagons_from_pixels(final_output_path, resolution=12):
+    """
+    Makes single geo points output with label output into a hexagon output.
+
+
+    @param final_output_path: path to a row pixelbased point data.
+    @param resolution: the resolution of the hexagon, look at https://h3geo.org/docs/core-library/restable/ for details
+    """
+
+    # TODO: Make clear exception that the data is not in order!
+    # if self.square_output:
```

Comment:
What are these self.square_output and self.dissolve_parts commented out bits here? They seem a bit out of context? Maybe remove them for clarity's sake.